src/main/java/uk/gov/ons/census/action/model/dto/FieldworkFollowup.java

```diff
@@ -31,4 +32,5 @@
   private Boolean undeliveredAsAddress;
   private Boolean blankQreReturned;
   private Boolean handDelivery;
+  private Map<String, String> metadata;
```

Comment:
Is it possible to use an Entity style class for this mapping instead of serialising all the data to strings and back?